java/ql/lib/semmle/code/java/frameworks/Servlets.qll

```diff
@@ -347,3 +347,27 @@ predicate isRequestGetParamMethod(MethodAccess ma) {
   ma.getMethod() instanceof ServletRequestGetParameterMapMethod or
   ma.getMethod() instanceof HttpServletRequestGetQueryStringMethod
 }
+
+/** The Java EE RequestDispatcher. */
+library class RequestDispatcher extends RefType {
```

Comment:
Oh, don't use `library`, it's deprecated.

```suggestion
class RequestDispatcher extends RefType {
```

Reply:
Removed